Add Comparison to Python Strings (#695)
Codecademy/docs: content/python/concepts/strings/strings.md

---
Title: 'Strings'

> Description: 'A string is a sequence of characters contained within a pair of single quotes or double quotes.'

Reviewer: Can we change to "single or double quotes" rather than "single quotes or double quotes"

## Accessing the Characters of a String

> Strings in Python are technically a type of list—one in which each character is a separate element in that list. This means each letter in a string can be accessed by indexing, as with the elements in a list:

Reviewer 2: ```suggestion
Strings in Python are technically a type of [list](https://www.codecademy.com/resources/docs/python/lists) — one in which each character is a separate element. This means each character in a string can be individually accessed by index, like with the elements in a list:
```

> ```codebyte/py

Reviewer: (See last comment) can we turn this into a regular code block?

  # Output: It's here!
```

## Comparing Strings

Python can use comparison [operators](https://www.codecademy.com/resources/docs/python/operators) to compare the contents of two strings. The operators behave as they do with numeric arguments:

- Equal, `==`, for returning `True` if two strings are equal.
- Not equal, `!=`, for returning `True` if two strings are not equal.
- Less than, `<`, for returning `True` if the left string is lexically prior to than the right string.
- Less than or equal to, `<=`, for returning `True` if the left string is equal to or lexically prior to than the right string.
- Greater than, `>`, for returning `True` if the left string comes lexically after the right string.

> - Greater than or equal to, `>=`, for returning `True` if the left string is equal to or comes lexically after the right string.

Reviewer: @Dusch4593 wondering what you think about having the operator col come first?

> ```py

Reviewer 2: To keep uniform with other Docs entries, the Codebyte snippet (if one can be used) should be the last snippet mentioned in the entry, located towards the bottom.

> ```py

Reviewer: Plus one on this! Docs users can copy/paste code to this codebyte on the bottom of the entries if desired.
